whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java

```diff
@@ -5026,6 +5026,538 @@ public void primary_key_comment() {
         assertThat(updateResponse.getErrorMessages().get(1).getAttribute(), is(new Attribute("nic-hdl", "PP1-TEST # update comment")));
     }
 
+    @Test
+    public void create_person_mnt_ref() {
+        final RpslObject createPerson = RpslObject.parse("" +
+                "person:    Pauleth Palthen\n" +
+                "address:   Singel 258\n" +
```

Comment:
If the test case is just for create person , can you go away with update [person test case here? Also can we use existing mntner it will make the test case smaller and easy to read